### PR TITLE
Final

### PR DESCRIPTION
**Pull request by melatran**
# Crud Functionality in Reviews

- Edit/Update/Delete functionality to reviews
- Only the user that created the review can see the edit/delete buttons
- test coverage at 97%

# Quiz Styling

- Quiz page was looking kinda sad so I added a neon sign type header to give it some pizzazz
- centered the table and the submit button for my sanity

### CODE REVIEW

File: app/assets/stylesheets/style.scss [hunk 1]
```diff
@@ -3605,14 +3605,66 @@ body {
...
+  }
+
+  0%,
+  19%,
+  21%,
+  23%,
+  25%,
+  54%,
+  56%,
+  100% {
```
Comment (melatran) [inline app/assets/stylesheets/style.scss:3661]: Adding some sparkle to the quiz page via title

Comment (whitneykidd) [reply app/assets/stylesheets/style.scss:3661]: This is awesome

File: README.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ To return the test suite to its normal state, change the connection address from
 
 ## Fear Quiz
 
-<img width="1427" alt="Screen Shot 2020-07-29 at 3 21 23 PM" src="https://user-images.githubusercontent.com/59414750/88854860-35c7f400-d1af-11ea-9317-3b41ed333c84.png">
+<img width="1402" alt="Screen Shot 2020-07-30 at 2 48 55 PM" src="https://user-images.githubusercontent.com/59414750/88973136-e4356d00-d273-11ea-922b-a23fc1f7a3ed.png">
```
Comment (perryr16) [inline README.md:91]: oh thats nice

File: app/views/game_search/show.html.erb [hunk 3]
```diff
@@ -59,6 +59,10 @@
                 <h5><%= review.title %></h5>
                 Rating: <%= review.user_rating %>
                 <p><%= review.content %></p>
+                <% if current_user && review.user_id == current_user.id %>
```
Comment (perryr16) [inline app/views/game_search/show.html.erb:62]: if it works it works

File: spec/features/reviews/edit_spec.rb [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    fill_in :content, with: "What are they feeding these pets?!"
+    fill_in :user_rating, with: 2
+
+    click_on "Update"
+
+    expect(current_path).to eq("/games/#{@game.id}")
+    expect(page).to have_content("What are they feeding these pets?!")
+  end
+
+  it "User can't update review if missing blanks" do
```
Comment (perryr16) [inline spec/features/reviews/edit_spec.rb:31]: nice sad path

**Pull request merged**